client/apps/user-portal/src/app/sermons/sermon-view/sermon-view.component.html

```diff
@@ -1 +1,58 @@
-<p>sermon-view works!</p>
+<div *ngIf="sermon$ | async as sermon">
+  <button mat-icon-button (click)="goBack()">
+    <mat-icon class="back-arrow" color="accent">arrow_back</mat-icon>
+  </button>
+
+  <div class="container-fluid">
+    <div class="d-flex justify-content-center">
+      <iframe
+        [src]="sanitizeUrl(sermon?.vimeoUrl)"
```

Comment:
- update this to get the video off of the "media" array in the model I suggested.
- remove the "vimeoUrl".
- make sure this checks if the media is a video, before giving it the URL.